packages/mobile/src/components/collection-list/CollectionList.tsx

```diff
@@ -112,6 +114,8 @@ const CollectionIDList = (props: CollectionIdListProps) => {
           source={createPlaylistSource!}
           sourceTrackId={createPlaylistTrackId}
           onCreate={createPlaylistCallback}
+          // TODO: support album type (we don't have use case currently)
```

Comment:
Just cause we don't have a create album with track?

Reply:
This is where this ticket's change will be: https://linear.app/audius/issue/C-3523/create-album-tile-on-profile-albums-mobile

It's the create playlist tile on the collection list